app/controllers/admin/splits_controller.rb

```diff
@@ -1,9 +1,26 @@
 class Admin::SplitsController < AuthenticatedAdminController
   def index
-    @splits = Split.active.order(:name)
+    if params[:app].present?
+      @app = App.find_by(name: params[:app])
+      return redirect_to admin_root_path unless @app
+
+      @splits = @app.splits
+    else
+      @splits = Split.all
+    end
+
+    @splits = @splits.active.order(sort_field)
   end
 
   def show
     @split = Split.find(params[:id])
   end
+
+  private
+
+  def sort_field
+    return :created_at if params[:sort] == 'date'
```

Comment:
thoughts on a configurable sort order as well? (asc vs desc)

Reply:
Didn't feel it was worth the hassle tbh - main goal here is to be able to see the oldest splits to know what we we can go after attacking. WDYT?

Reply:
Yeah that seems reasonable to me. There's some UI component reworking I'd love to do so that we'd get more of these features for free anyways. Happy to leave that for later.